qutebrowser/browser/browsertab.py

```diff
@@ -981,9 +992,31 @@ def run_js_async(self, code, callback=None, *, world=None):
     def shutdown(self):
         raise NotImplementedError
 
-    def title(self):
+    def _title(self):
         raise NotImplementedError
 
+    def title(self, *, fallback=True):
+        """Get the title of the webpage.
+
+        Args:
+            fallback: If set, use an override or fallback title
```

Comment:
This is a bit ambiguous, it's not clear if 'set' means 'True' (as setting it to true) or 'False' (as overriding the default).

This feels a bit weird to me because 'fallback=False' is the only case where 'self.data.title_override' is used (which seems like a fallback to me). Maybe the values are reversed?

Reply:
Err, yes, that's supposed to say `and fallback`... I'll fix that and try to clarify the docs a bit.